erdpy/testnet/node_config_toml.py

```diff
@@ -11,12 +11,15 @@ def patch(data: Any, testnet_config: TestnetConfiguration):
     data['GeneralSettings']['PenalizedTooMuchGasEnableEpoch'] = 0
     data['GeneralSettings']['SwitchJailWaitingEnableEpoch'] = 0
     data['GeneralSettings']['BelowSignedThresholdEnableEpoch'] = 0
-    data['GeneralSettings']['SwitchHysteresisForMinNodesEnableEpoch'] = 0
+    data['GeneralSettings']['SwitchHysteresisForMinNodesEnableEpoch'] = 1
     data['GeneralSettings']['TransactionSignedWithTxHashEnableEpoch'] = 0
     data['GeneralSettings']['MetaProtectionEnableEpoch'] = 0
     data['GeneralSettings']['AheadOfTimeGasUsageEnableEpoch'] = 0
     data['GeneralSettings']['GasPriceModifierEnableEpoch'] = 0
 
+    # Make epochs shorter
+    data['EpochStartConfig']['RoundsPerEpoch'] = 50
```

Comment:
I think 100 is the minimum we have tried and found out is working properly

Reply:
Oh. Good to know. Will set it to 100.